src/ui/components/Library/Team/TeamContextRoot.tsx

```diff
@@ -9,7 +9,13 @@ import { useGetWorkspace } from "ui/hooks/workspaces";
 
 import { LibrarySpinner } from "../LibrarySpinner";
 
-export const MY_LIBRARY_TEAM = { name: "Your Library", isTest: false, id: "me", databaseId: null };
+export const MY_LIBRARY_TEAM = {
+  name: "Your Library",
+  isTest: false,
+  retentionLimit: 7,
```

Comment:
Let's set this to `null` for now since personal recordings do not have a retention period right now.